lib/fathead/javascript_eslint/doc/notes.md

```diff
@@ -0,0 +1,52 @@
+# Notes
```

Comment:
@moollaza thx for your quick reply. I took a look at the other fatheads in this repo. None of them had a MD doc at top-level, so I moved to doc/.

No problem, I'll move that.